edx_reverification_block/xblock/reverification_block.py

```diff
@@ -29,6 +30,20 @@ def student_view(self, context=None):
         course_id = self.get_course_id()
         item_id = unicode(self.scope_ids.usage_id)
         checkpoint_name = CHECKPOINT_NAME
+        # TODO: How to get the user object from runtime to check the status of that user in the given checkpoint?
```

Comment:
We won't be able to access the `User` object, since the XBlock may be installed in a runtime that isn't implemented in Django.  However, we can get the username like this:
```
username = unicode(self.scope_ids.user_id)
```
We can then pass the username to the `VerificationService`, which can look up the User object.